### ISSUE TITLE
[Feature Request] Copy languages

### ISSUE THREAD

**Issue by JCKodel** [FriendsOfFlarum/linguist#12]
Sometimes, we just want to see admin plugins messages in plain english (instead of translation key - the default when no translation is available).

Would be awesome a button to "Copy all [english] text into empty [brazilian portuguese] keys".

Otherwise, we have to manually copy and paste each entry to make a non translated extension to work:

![image](https://user-images.githubusercontent.com/379339/65294338-adaba380-db34-11e9-9d18-2715ae1b9b65.png)

**Comment by clarkwinkelmann** [comment]
While this would be possible, I still think fixing the underlying Flarum issue is a priority https://github.com/flarum/core/issues/1774

When that issue is fixed, copying the whole language will no longer make sense.

**Comment by clarkwinkelmann** [comment]
Since translation fallback is now working as expected in Flarum (since several releases already), I'm going to close this issue as I don't see any use case here.
